src/manage/commands.py

```diff
@@ -476,73 +514,62 @@ def execute(self):
     def usage_text_lines(cls):
-        usage_docs = [
-            (f"    {EXE_NAME} -V:!B!<TAG>!W!",
-             "Launch runtime identified by !B!<TAG>!W!, which should include the " +
-             "company name if not !B!PythonCore!W!. Regular Python options may " +
-             "follow this option."),
-            (f"    {EXE_NAME} -!B!<VERSION>!W!",
-             r"Equivalent to -V:PythonCore\!B!<VERSION>!W!. The version must " +
-             "begin with the digit 3, platform overrides are permitted, " +
-             "and regular Python options may follow." +
-             (" !G!py -3!W! is the equivalent of the !G!python3!W! command." if EXE_NAME == "py" else "")),
-            (f"    {EXE_NAME} !B!<COMMAND>!W!",
-             "Run a specific command (see list below)."),
-        ]
-
-        usage_ljust = max(len(logging.strip_colour(i[0])) for i in usage_docs)
+        if EXE_NAME.casefold() in ("py".casefold(), "pyw".casefold()):
+            usage_docs = PY_USAGE_DOCS
+        else:
+            usage_docs = PYMANAGER_USAGE_DOCS
+
+        usage_docs.extend(
+            [
+                (
+                    f"{EXE_NAME} " + getattr(COMMANDS[cmd], "USAGE_LINE", cmd),
+                    getattr(COMMANDS[cmd], "HELP_LINE", "")
+                )
+                for cmd in sorted(COMMANDS)
+                if cmd[:1].isalpha()
+            ]
+        )
+
+        usage_docs = [("    " + x.lstrip(), y) for x, y in usage_docs]
+
+        usage_ljust = max(len(logging.strip_colour(i[0])) for i in usage_docs if not i[0].endswith("\n"))
         if usage_ljust % 4:
             usage_ljust += 4 - (usage_ljust % 4)
         usage_ljust = max(usage_ljust, 16) + 1
         sp = " " * usage_ljust
 
         yield "!G!Usage:!W!"
-        if EXE_NAME.casefold() in ("py".casefold(), "pyw".casefold()):
-            yield f"    {EXE_NAME} !B!<regular Python options>!W!"
-            yield sp + "Launch the default runtime with specified options."
-            yield sp + "This is the equivalent of the !G!python!W! command."
         for k, d in usage_docs:
-            r = k.ljust(usage_ljust + len(k) - len(logging.strip_colour(k)))
+            if k.endswith("\n") and len(logging.strip_colour(k)) >= usage_ljust:
+                yield k.rstrip()
+                r = sp
+            else:
+                k = k.rstrip()
+                r = k.ljust(usage_ljust + len(k) - len(logging.strip_colour(k)))
             for b in d.split(" "):
-                if len(r) >= 80:
+                if len(r) >= logging.CONSOLE_MAX_WIDTH:
                     yield r.rstrip()
                     r = sp
                 r += b + " "
             if r.rstrip():
                 yield r
 
         yield ""
-        yield "Find additional information at !B!https://docs.python.org/using/windows.html!W!."
+        # TODO: Remove the 3.14 for stable release
+        yield "Find additional information at !B!https://docs.python.org/3.14/using/windows!W!."
```

Comment:
The link techincally ends with `.html`. Perhaps also use the `/dev/` symlink if you don't want to tie to 3.14?

[`https://docs.python.org/dev/using/windows.html`](https://docs.python.org/dev/using/windows.html)

Reply:
It redirects fine without the `.html`, and it's cleaner (for a user-visible link). I forgot about `dev` though, thanks